agent/intents/hwrks_usersays_en.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "7835deaa-9783-4ca8-a9e7-c7cb849136f2",
+    "data": [
+      {
+        "text": "hwrks?",
+        "userDefined": false
+      }
+    ],
+    "isTemplate": false,
+    "count": 0
+  },
+  {
+    "id": "a853160b-35e9-4594-8c86-86bbac683ad0",
+    "data": [
+      {
+        "text": "hwrks",
+        "userDefined": false
+      }
+    ],
+    "isTemplate": false,
+    "count": 0,
+    "updated": 1515902978
+  }
```

Comment:
This is natural language processing. I dont think so that you need to add short form as a way to add command. Just think of how normal user would be addressing the bot and add commands like `How does this work` and `Tell me how this works.`